GW2EIEvtcParser/EncounterLogic/Strikes/EndOfDragons/HarvestTemple.cs

```diff
@@ -812,25 +907,16 @@ internal override void ComputePlayerCombatReplayActors(AbstractPlayer p, ParsedE
                 knownEffectsIDs.Add(redSelected.ContentID);
                 foreach (EffectEvent redSelectedEffect in redSelectedEffects)
                 {
-                    int duration = 6500;
+                    int duration = 7000;
                     int start = (int)redSelectedEffect.Time;
                     int end = start + duration;
                     AbstractSingleActor dragonVoid = FindActiveOrNextDragonVoid(redSelectedEffect.Time);
                     if (dragonVoid == null)
                     {
                         continue;
                     }
-                    int puddleEnd = Math.Min((int)dragonVoid.LastAware, end + 300000); // puddles stay alive for 5 minutes
-                    int effectEnd = Math.Min(puddleEnd, end);
-                    replay.Decorations.Add(new CircleDecoration(true, end, 400, (start, effectEnd), "rgba(250, 50, 0, 0.2)", new AgentConnector(p)));
-                    replay.Decorations.Add(new CircleDecoration(true, 0, 400, (start, effectEnd), "rgba(250, 50, 0, 0.2)", new AgentConnector(p)));
-                    Point3D pos = p.GetCurrentPosition(log, end);
-                    if (pos == null)
-                    {
-                        continue;
-                    }
-                    replay.Decorations.Add(new CircleDecoration(true, end + 1500, 400, (end, puddleEnd), "rgba(250, 0, 0, 0.3)", new PositionConnector(pos)));
-                    replay.Decorations.Add(new CircleDecoration(true, 0, 400, (end, puddleEnd), "rgba(250, 0, 0, 0.3)", new PositionConnector(pos)));
+                    replay.Decorations.Add(new CircleDecoration(true, end, 400, (start, end), "rgba(250, 50, 0, 0.2)", new AgentConnector(p)));
```

Comment:
End should be majored by the last aware of the dragonVoid like before, as if the selection happens before phasing, the effect ends immediately when phase end.